python/ray/util/spark/data.py

```diff
@@ -0,0 +1,46 @@
+import ray
+from urllib.parse import urlparse
+from ray.util.spark.utils import get_spark_session, is_in_databricks_runtime
+
+
+def _convert_dbfs_path_to_local_path(dbfs_path):
+    parsed_path = urlparse(dbfs_path)
+    assert parsed_path.scheme.lower() == "dbfs", "dbfs path is required."
+    return "/dbfs" + parsed_path.path
+
+
+def load_spark_dataset(path, saved_format):
+    """
+    Returns a Ray dataset that loads data from a saved spark dataset path.
+
+    Args:
+        path: The path to the saved spark dataset.
+        saved_format: The format of the saved dataset, only 'parquet' and
+            'delta' format are supported. 'delta' format is only supported
+            on Databricks runtime.
+    """
+    spark = get_spark_session()
+    assert saved_format in [
+        "parquet",
+        "delta",
+    ], f"Unsupported saved format {saved_format}"
```

Comment:
```suggestion
    assert saved_format in [
        "parquet",
        "delta",
    ], f"Unsupported saved format {saved_format}"
    spark = get_spark_session()
```

Reply:
I am considering to automatically infer the saved data type.